specs/protocol/revenue-sharing.md

```diff
@@ -0,0 +1,48 @@
+# Revenue Sharing
+
+Revenue sharing is the process by which chains in the superchain contribute a portion of their revenue or profit to the Optimism collective. They do this in return for support with the OP Stack and other benefits. 
+
+
+## Definitions
+| Term   | Name         | Definition  |
+| -------|--------------| ----------- |
+| $d$ | L1 Data Fee Revenue    | ETH transferred to the sequencer with L2 transactions to cover estimated L1 Data Fees(see below). Accumulates to `L1FeeVault`.
+| $e$ | L1 Data Fee Expenditure| ETH spent by the batcher on L1 to make transaction data available
+| $b$ | L2 Base Gas Revenue         | Base fee portion of L2 Gas Fee (ETH transferred to the sequencer to cover execution of L2 transactions). Accumulates to `BaseFeeVault`.
+| $p$ | L2 Priority Gas Revenue         | Priority fee portion of L2 Gas Fee (ETh transferred to the sequencer to cover execution of L2 transactions). Accumulates to `SequencerFeeVault`.
+| $g$ | L2 Gas Revenue         | $b+p$
+| $r$ | Sequencer Revenue      | $e + g$
+| $p$ | Sequencer Profit       | $e + g - d$
+| $s$ | Revenue share due to Optimism Collective | $\max(0.15r,0.025p)$
+
+## `RevenueSharer` predeploy
+Revenue sharing is achieved through an L2 contract `RevenueSharer`. This is the implementation of a proxy contract `RevenueSharerProxy` with address
+
+```
+0x4200000000000000000000000000000000000022
+```
+
+### Deploying `RevenueSharer`
+TODO
+
+### Execution
+Revenue sharing is executed periodically. 
+The `RevenueSharer` is respomsible for computing $s$ and sending it to a predetermined address controlled by the Optimism Collective. At the end of execution, `SequencerFeeVault`, `L1FeeVault` and `BaseFeeVault` and `RevenueSharer(Proxy)` should be completely depleted of ETH. This allows the contract to be stateless.
```

Comment:
Calling it with the proxy suffix would also require calling each of the others with the proxy suffix, i think its best to just drop the proxy suffix completely

Reply:
Good call, fixed in [1cbe6bd](https://github.com/ethereum-optimism/specs/pull/115/commits/1cbe6bd0b64993de125df6ffce0afceb31eb9ab7)